__tests__/endpoints.test.js

```diff
@@ -110,13 +110,64 @@ describe('GET /api/articles', () => {
           expect(article).toHaveProperty('article_img_url');
           expect(article).toHaveProperty('comment_count')
           expect(article).not.toHaveProperty('body')
-        });                
+        })
+      });
+  });
+});
+        
+describe('GET /api/articles/:article_id/comments', () => {
+  test("Status 200: Should respond with comments for an article filtered by article_id order by latest comments", () => {
+    return request(app)
+      .get('/api/articles/6/comments')
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body)).toBe(true);
+        expect(body).toBeSorted({ descending: true, key: 'created_at' });
+        body.forEach((comment) => {
+          expect(comment).toHaveProperty('comment_id');
+          expect(comment).toHaveProperty('votes');
+          expect(comment).toHaveProperty('created_at');
+          expect(comment).toHaveProperty('author');
+          expect(comment).toHaveProperty('body');
+          expect(comment).toHaveProperty('article_id');
+        });
       });
   });
 });
 
+describe('GET /api/articles/:article_id/comments', () => {
+  test("Should respond with 404 Not found for an article_id that does not exist", () => {
+    return request(app)
+      .get('/api/articles/76/comments')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not found')
+      });
+      });
+});
 
+describe('GET /api/articles/:article_id/comments', () => {
+  test("Should respond with 404 Not found for an  article_id that exists but does not have any comments", () => {
+    return request(app)
+      .get('/api/articles/10/comments')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not found')
+      });
```

Comment:
Would this be a 404? If the article ID exists but does not have any comments, we would expect a 200 with an empty comments array 👍